src/devtools/useAtomDevtools.ts

```diff
@@ -4,21 +4,50 @@ import type { Atom, WritableAtom } from 'jotai'
 import type { Scope, SetAtom } from '../core/atom'
 import { Message } from './types'
 
+interface DevtoolOptions {
+  name?: string
+  scope?: Scope
+  enabled?: boolean
+}
+export function useAtomDevtools<Value, Result extends void | Promise<void>>(
+  anAtom: WritableAtom<Value, Value, Result> | Atom<Value>,
+  options?: DevtoolOptions
+): void
+
+/*
+ * @deprecated Please use object options (DevtoolsOptions)
+ */
+export function useAtomDevtools<Value, Result extends void | Promise<void>>(
+  anAtom: WritableAtom<Value, Value, Result> | Atom<Value>,
+  options?: string,
+  scope?: Scope
+): void
+
 export function useAtomDevtools<Value, Result extends void | Promise<void>>(
   anAtom: WritableAtom<Value, Value, Result> | Atom<Value>,
-  name?: string,
+  options?: DevtoolOptions | string,
   scope?: Scope
```

Comment:
```suggestion
  deprecatedScope?: Scope
```
changing like that make code clearer.